docs/sdk/SDK_API_DEFINITION.md

```diff
@@ -0,0 +1,205 @@
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_apply_connection_config_video()
+```c
+int mesh_apply_connection_config_video(MeshConnection *conn,
+                                       MeshConfig_Video *cfg)
+```
+Applies the configuration to setup the connection payload for Video frames
+
+#### Parameters
+* `[IN]` `conn` – Pointer to a connection structure.
+* `[IN]` `cfg` – Pointer to a configuration structure.
+
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_apply_connection_config_audio()
+```c
+int mesh_apply_connection_config_audio(MeshConnection *conn,
+                                       MeshConfig_Audio *cfg)
+```
+Applies the configuration to setup the connection payload for Audio packets
+
+#### Parameters
+* `[IN]` `conn` – Pointer to a connection structure.
+* `[IN]` `cfg` – Pointer to a configuration structure.
+
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_delete_connection()
+```c
+int mesh_delete_connection(MeshConnection **conn)
+```
+Deletes the connection and its resources.
+
+#### Parameters
+* `[IN/OUT]` `conn` – Address of a pointer to the connection structure.
+
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_get_buffer()
+```c
+int mesh_get_buffer(MeshConnection *conn,
+                    MeshBuffer **buf)
+```
+Gets a buffer from the media connection.
+
+#### Parameters
+* `[IN]` `conn` – Pointer to a connection structure.
+* `[OUT]` `buf` – Address of a pointer to a mesh buffer structure.
+
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_get_buffer_timeout()
+```c
+int mesh_get_buffer_timeout(MeshConnection *conn,
+                            MeshBuffer **buf,
+                            int timeout_ms)
+```
+Gets a buffer from the media connection with a timeout.
+
+#### Parameters
+* `[IN]` `conn` – Pointer to a connection structure.
+* `[OUT]` `buf` – Address of a pointer to a mesh buffer structure.
+* `[IN]` `timeout_ms` – Timeout interval in milliseconds.
+
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_put_buffer()
+```c
+int mesh_put_buffer(MeshBuffer **buf)
+```
+Puts the buffer to the media connection.
+
+#### Parameters
+* `[IN/OUT]` `buf` – Address of a pointer to a mesh buffer structure.
+
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_put_buffer_timeout()
+```c
+int mesh_put_buffer_timeout(MeshBuffer **buf,
+                            int timeout_ms)
+```
+Puts the buffer to the media connection with a timeout.
+
+#### Parameters
+* `[IN/OUT]` `buf` – Address of a pointer to a mesh buffer structure.
+* `[IN]` `timeout_ms` – Timeout interval in milliseconds.
+
+#### Returns
+0 if successful. Otherwise, returns an error.
+
+
+## mesh_err2str()
+```c
+const char *mesh_err2str(int err)
+```
+Gets a text description of the error code.
+
+#### Parameters
+* `[IN]` `err` – Error code returned from any Mesh DP API call.
```

Comment:
what is "DP"? "Data Plane" ? Define this abbreviation somewhere.

Reply:
Fixed in eae8b083e0210567994eb379b5e228742451c44b.